dockstore-webservice/src/main/java/io/dockstore/webservice/core/Author.java

```diff
@@ -101,4 +103,22 @@ public String getEmail() {
     public void setEmail(final String email) {
         this.email = email;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Author)) {
```

Comment:
fwiw, the new `instanceof` is pretty cool, you can do:
```
if (!(o instanceof Author that)) {
```
and `that` will be of type `Author` and in scope for the rest of the function outside the `if` block (I think, didn't test it).